### PR TITLE
Setup alternate Capsule ports

### LINKED ISSUE #380: Setup alternate Capsule ports
Setup alternate Capsule ports
* added setup_alternate_capsule_ports() superseding enable_gateway_ports_connections()
- related to https://github.com/SatelliteQE/robottelo/issues/3453 (merge together)

We are going to fix it by labeling predefined range of ports (9091-9190) as websm_port_t ports. Plus we dont need ssh for tunneling instead we will switch to use socat tool

### PR DESCRIPTION
**Pull request by lpramuk**
* added setup_alternate_capsule_ports() superseding enable_gateway_ports_connections()
* related to SatelliteQE/robottelo#3453 (merge together)

(Fixes #380)

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1314,7 +1314,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             host=host
         )
     execute(fix_qdrouterd_listen_to_ipv6, host=host)
-    execute(enable_gateway_ports_connections, host=host)
```
Comment (elyezer) [inline automation_tools/__init__.py]: Since the `enable_gateway_ports_connections` will not be used anymore, please remove that task.

Comment (lpramuk) [inline automation_tools/__init__.py]: OK, I'll do it
I thought that it could be reused for some other purposes in future...

**Comment by lpramuk** [comment]
Elyezer's comment resolved.

**Comment by lpramuk** [comment]
I changed the task once more to prepare ```nc``` instead of ```socat``` as we chose to use ```nc -klp <alternate_port> -c "nc 127.0.0.1 9090"``` for robottelo's capsule "mocking"

**Comment by elyezer** [comment]
ACK

**Comment by lpramuk** [comment]
No longer waits as Robottelo PR#3453 is also ready to merge

**Comment by rplevka** [comment]
ACK

**Pull request merged**